src/PhpSpreadsheet/Writer/Xlsx/Chart.php

```diff
@@ -1127,9 +1151,6 @@ private function writePlotGroup($plotGroup, $groupType, $objWriter, &$catIsMulti
                 $objWriter->endElement();
             }
 
-            //    Values
-            $plotSeriesValues = $plotGroup->getPlotValuesByIndex($plotSeriesRef);
```

Comment:
Why was this removed ?

Reply:
It was moved to line 1100 ($plotSeriesValues is needed earlier).